Creative design ideas

```diff
@@ -27,4 +27,12 @@
  "id": 6,
  "Description": "An app that contains various customizable BookMarks according to the requirement",
 },
+{
+  "id" : 7,
+  "Description" : "An app that shows the price/cost from all three or more like Uber, Ola, Rapido so that we can choose the least one."
```

Comment:
"Description" : "An app that shows the price/cost from all three or more like Uber, Ola, Rapido so that we can choose the least one.",